client/gutenberg/editor/edit-post/components/header/index.js

```diff
@@ -66,7 +63,9 @@ function Header( {
 						aria-expanded={ isEditorSidebarOpened }
 					>
 						<DotTip id="core/editor.settings">
-							{ __( 'You’ll find more settings for your page and blocks in the sidebar. Click ‘Settings’ to open it.' ) }
+							{ __(
```

Comment:
If this ends up not being published from Gutenberg, should we switch this to a translate call from our i18n package?

As an aside, @Automattic/team-calypso are we planning on standardizing to `@wordpress/i18n` or will we keep [i18n-calypso](https://github.com/Automattic/i18n-calypso) ?